homebrewing/src/components/Brewing/Brewing.jsx

```diff
@@ -11,12 +11,19 @@ export default function Brewing() {
 const currentWidth = window.innerWidth
 
 useEffect(() => {
+
   const resizeListener = () => {
     setWidth(window.innerWidth);
   };
+
   window.addEventListener("resize", resizeListener);
   window.addEventListener('load', resizeListener);
   setWidth(currentWidth);
+
+  return () => {
+    window.removeEventListener("resize", resizeListener);
+    window.removeEventListener('load', resizeListener);
+  }
```

Comment:
this is nice, just wondering if this was part of the original plan to include this change in the prettier work? It's just semantics but worth asking

Reply:
Ahh I see what is happening, I think you were branching off a feature branch hence these include commits from the other PRs - we may need to speak about that to help you manage your feature work better.

Reply:
Oh I see, I sily mixed with this branch and other branch maybe in my VScode when Im pushing or pulling, I missed somthing...oh...it's not good.